src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -21,13 +22,13 @@ sealed class ConfigurationFeatureDefinitionProvider : IFeatureDefinitionProvider
         // IFeatureDefinitionProviderCacheable interface is only used to mark this provider as cacheable. This allows our test suite's
         // provider to be marked for caching as well.
 
-        private const string FeatureFiltersSectionName = "EnabledFor";
-        private const string RequirementTypeKeyword = "RequirementType";
         private readonly IConfiguration _configuration;
         private readonly ConcurrentDictionary<string, FeatureDefinition> _definitions;
         private IDisposable _changeSubscription;
         private int _stale = 0;
 
+        const string ParseValueErrorString = "Invalid {0} with value '{1}' for feature '{2}'.";
```

Comment:
How come `{0}` isn't in single quotes ? And shouldn't it be "Invalid settting '{0}' with value..."

Reply:
I just based the format string off of the old error message, but since we're using a variable now I understand adding the single quotes.